src/app/public/modules/sort/sort.component.ts

```diff
@@ -28,14 +28,18 @@ let nextId = 0;
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SkySortComponent {
-  public sortByHeadingId: string = `sky-sort-heading-${++nextId}`;
 
+  /**
+   * Indicates whether to display text on the sort button beside the sort icon.
```

Comment:
Should we specify what the text is? Since this is a boolean and we don't have an accompanying property for specifying the text, it seems like we have a default label that this turns on. And looking at the demo, I assume that default label is `Sort`. Is that correct? And if so, should we update this description to make that clearer? I'm thinking `Indicates whether to display a **Sort** label beside the sort icon on the sort button.` What do you think?

Reply:
Yep, I likey. Done.

Reply:
Looks like you implemented this, but I modified it, putting quote marks around "Sort" to match the similar previous item for a filter button and to remove "sort" before "icon" to avoid repeating the word three times.